src/components/Form.tsx

```diff
@@ -54,17 +54,16 @@ export const Form = ({
   };
 
   const Buttons = (): JSX.Element => {
-    function save(): void {
+    async function save(): Promise<void> {
       if (selectedOption) {
-        Create(selectedOption)
-          .then(() => {
-            closeForm();
-            setIsSubButtonsVisible(false);
-            showAlert();
-          })
-          .catch((err: Error) => {
-            console.log(err); // TODO handle this by using alert
-          });
+        try {
+          await Create(selectedOption);
+          closeForm();
+          setIsSubButtonsVisible(false);
+          showAlert();
+        } catch (error) {
+          console.log(error); // TODO handle this by using alert
+        }
       }
```

Comment:
how about?
```
if (!selectedOption) return;
try {
          await Create(selectedOption);
          closeForm();
          setIsSubButtonsVisible(false);
          showAlert();
} catch (error) {
          console.log(error); // TODO handle this by using alert
}
```

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/8/commits/233f83819cfd89e6b23370c50da0ba2e394eb4f1, thank you